src/java.desktop/share/classes/javax/swing/plaf/nimbus/skin.laf

```diff
@@ -21578,7 +21578,7 @@
                            <visible>true</visible>
                            <shapes>
                               <!--<rectangle x1="3.9545454545454546" x2="6.0" y1="26.49999999999998" y2="10.772727272727275" rounding="2.000000000000001">-->
-                                 
+
```

Comment:
Is this newline needed?

Reply:
No , it automatically got added.Need to remove this.

Reply:
Done